applications/band-grants.md

```diff
@@ -0,0 +1,164 @@
+
+Below we provide an **example roadmap**.
+
+> :exclamation: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+
+- **Total Estimated Duration:** 3-4 months
+- **Full-Time Equivalent (FTE):**  2
+- **Total Costs:** 80,000 USD
+
+### Milestone 
+
+*NB: do we need to add infra fees?*
+
+- **Estimated duration:** 3-4 months
+- **FTE:**  2
+- **Costs:** 80,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+|  Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide inline documentation of the code, a basic tutorial on how to use the smart contract and an example consumer contract that shows a practical example on how to use the smart contract|
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests.|
+| **0d.** | Docker | N/A for smart contracts|
+| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.) |
+| 1. | Data Source | Data sources on BandChain which will be the core of getting data that will be delivered to Astar |
+| 2. | Oracle Script | An oracle script on BandChain which will verify and aggregate data from the data sources on BandChain|
+| 3. | BandChain Requester | A service that will prepare and request data from BandChain to be sent to Astar | 
+| 4. | Astar ink! Relayer | A service that will prepare and execute data to be bridged over to Astar|
+| 5. | ink! Signer | A seperate but crucial service that will used concurrently to Astar to send over data |
+| 6. | PDS | A service on BandChain that will allow paid services to deliver data to BandChain safely and securely |
+| 7. | Smart contract | An ink! smart contract that will allow consumers to query various price feed from BandChain|
+| 8. | Testing and Audit | Testing|
+| 9. | Maintenance | Reviewer can continuously read the state of the contract to get an asset's price|
+| 10. | Infrastructure | Hosting cost - Storage, GCP|
+| 11. | Monitoring | 24/7 monitoring service ensures the continuous operation of the relayer. This includes the comprehensive monitoring of both the Astar EVM and Astar WASM chains for a duration of one year|
```

Comment:
Do you expect to expand the oracle service to other ink! based blockchain like AlephZero?

Reply:
We are excited about the opportunity to expand our oracle service to additional ink!-based blockchains. However, it is important to clarify that the grant we are seeking will be exclusively allocated to support the development and maintenance of Astar WASM.

Reply:
Updated, see changes in line 56